rust/windows-client/src-tauri/src/client.rs

```diff
@@ -89,7 +89,7 @@ pub(crate) fn run() -> Result<()> {
         Some(Cmd::CrashHandlerServer { socket_path }) => crash_handling::server(socket_path),
         Some(Cmd::Debug { command }) => debug_commands::run(command),
         // If we already tried to elevate ourselves, don't try again
-        Some(Cmd::Elevated) => run_gui(cli),
+        Some(Cmd::Elevated | Cmd::SmokeTest) => run_gui(cli),
```

Comment:
I'd rather we inline the function here so we can split off this pattern matching, I think it's clearer to follow the flow of this function.